misc/network.c

```diff
@@ -737,6 +737,9 @@ socket_negotiate_ssl (int fd, openvas_encaps_t transport,
  * @param[out]  cert    Memory pointer to fill cert pointer.
  * @param[out]  certlen Size of cert.
  */
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic warning "-Wdeprecated-declarations"
```

Comment:
`#pragma GCC diagnostic pop` missing after the g_memdup call. 
And I would only enclose the `g_memdup` call and not the complete function with the pragma.
Same for all the other uses.

Reply:
After enclosing just the call it doesn't compile; therefore will leave it on function level.